src/test/java/com/puppycrawl/tools/checkstyle/internal/CommitValidationTest.java

```diff
@@ -103,6 +103,28 @@ public static void setUp() throws Exception {
     }
 
     @Test
+    public void testHasCommits() {
+        assertTrue("must have atleast one commit to validate",
+                lastCommits != null && !lastCommits.isEmpty());
+    }
+
+    @Test
+    public void testCommitMessage() {
+        assertTrue("should not accept commit message with periods on end",
+                !ACCEPTED_COMMIT_MESSAGE_PATTERN.matcher("minor: Test. Test.").matches());
+        assertTrue("should not accept commit message with spaces on end",
+                !ACCEPTED_COMMIT_MESSAGE_PATTERN.matcher("minor: Test. ").matches());
+        assertTrue("should not accept commit message with tabs on end",
+                !ACCEPTED_COMMIT_MESSAGE_PATTERN.matcher("minor: Test.\t").matches());
+        assertTrue("should not accept commit message with newline on end",
+                !ACCEPTED_COMMIT_MESSAGE_PATTERN.matcher("minor: Test.\n").matches());
+        assertTrue("should not accept commit message with missing prefix",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [assertTrue(!expr) can be replaced by assertFalse(expr)](https://www.codacy.com/app/checkstyle/checkstyle/file/2579648286/issues/source?bid=2193633&fileBranchId=3302361#l121)